crew

```diff
@@ -697,6 +697,7 @@ def build_package (pwd)
 
   # build package from filelist, dlist and binary files in CREW_DEST_DIR
   puts "Archiving..."
+  install_package pwd
```

Comment:
I don't think we need this, do we?  Currently, crew works as expected without doing this.

Reply:
That wasn't supposed to make it into this PR. It would be nice to have a 'crew build --install <package_name' (spoiler: that doesn't work.)